models/point_of_interest.py

```diff
@@ -1,13 +1,24 @@
+from typing import List
 from pydantic import BaseModel
 
 
 class PointOfInterest(BaseModel):
-    name: str = ""
-    country: str = ""
-    city: str = ""
-    address: str = ""
-    lat: str = ""
-    lng: str = ""
-    categories: str = ""
-    organizations: str = ""
-    description: str = ""
+    name: str = ''
+    country: str = ''
+    city: str = ''
+    address: str = ''
+    categories: List[str] = []
```

Comment:
Make sure that these convert correctly into CSV and don't break existing unit tests for spiders

Reply:
The tests I did fix, but export to CSV might actually work a bit differently. Let me poke around it a bit more…